src/vues/contenant.js

```diff
@@ -1,36 +1,23 @@
-/* global alert */
-
-class VueContenant {
-  constructor (topologie) {
+export class VueContenant {
+  constructor (topologie, elementContenants) {
     this.topologie = topologie;
+    this.elementContenants = elementContenants;
   }
 
-  createElement (contenant, dimentionsEtageres) {
-    const vue = this;
+  affiche (contenant, cbOuvrirContenant) {
+    const width = this.topologie.contenants[contenant.type][contenant.categorie].largeur;
```

Comment:
Peut-être pourrait-on éviter d'exposer la structure interne de `topologie` en l'encapsulant dans une classe, et en appelant `this.topologie.largeur(contenant.type, contenant.categorie)` ?

Reply:
C'est une bonne idée. Comme on en a discuté, cela fera l'objet d'une PR séparée.